rubrik_cdm/cloud.py

```diff
@@ -618,14 +619,13 @@ def update_aws_native_account(self, aws_account_name, config, timeout=15):
         Returns:
             dict -- The full API response for `PATCH /aws/account/{id}'`.
```

Comment:
Is this supposed to be:

dict -- The full API response for `PATCH /aws/account/{id}`. 

Looks like there's a hanging ' in there.

Reply:
Updated.